packages/cli/src/config/settingsSchema.ts

```diff
@@ -341,6 +341,16 @@ export const SETTINGS_SCHEMA = {
     description: 'Command to start an MCP server.',
     showInDialog: false,
   },
+  shellToolRcFile: {
```

Comment:
Definitely a nit, but putting this between two mcp configs in the schema seems odd. There doesn't seem to be an obvious order in this file in general though, so it's probably fine.